runtime/kusama/src/xcm_config.rs

```diff
@@ -172,6 +172,7 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 			RuntimeCall::Timestamp(..) |
 			RuntimeCall::Indices(..) |
 			RuntimeCall::Balances(..) |
+			RuntimeCall::Crowdloan(..) |
```

Comment:
Unfortunately, we can't add every extrinsic from the crowdloan pallet to this list -- we must omit the `add_memo` extrinsic.

```suggestion
			RuntimeCall::Crowdloan(
				crowdloan::Call::create { .. } |
				crowdloan::Call::contribute { .. } |
				crowdloan::Call::withdraw { .. } |
				crowdloan::Call::refund { .. } |
				crowdloan::Call::dissolve { .. } |
				crowdloan::Call::edit { .. } |
				crowdloan::Call::poke { .. } |
				crowdloan::Call::contribute_all { .. }
			) |
```